extensions/pkg/controller/infrastructure/reconciler.go

```diff
@@ -230,7 +230,7 @@ func (r *reconciler) restore(ctx context.Context, infrastructure *extensionsv1al
 		return reconcile.Result{}, fmt.Errorf("%s: %+v", msg, err)
 	}
 
-	err := r.updateStatusSuccess(ctx, infrastructure, gardencorev1beta1.LastOperationTypeMigrate, EventInfrastructureRestoration, "Successfully restored infrastructure")
+	err := r.updateStatusSuccess(ctx, infrastructure, operationType, EventInfrastructureRestoration, "Successfully restored infrastructure")
```

Comment:
Is the `operationType` passed to restore func is `LastOperationTypeRestore`. Checking the `ComputeOperationType` func, it is not the case, I think. Don't we want to set it to `LastOperationTypeRestore` instead?

Reply:
@vlvasilev

Reply:
There was no `LastOperationTypeRestore` and at the time this PR been made there was no need for it. Simply the addition of this constant would serve only as information string back then.

Reply:
What about now? Do we want to use `LastOperationTypeRestore` or use `operationType` which I think is something different from `LastOperationTypeRestore`?

Reply:
Generally we want to use LastOperationTypeRestore. I'll adapt the PR accordingly but it might take some time to test it out.